### PR TITLE
Updates to new client library and fixes broken link in README

### PR DESCRIPTION
**Pull request by gguuss**

### CODE REVIEW

File: language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java [hunk 1]
```diff
@@ -157,29 +143,16 @@ public static void printSyntax(PrintStream out, List<Token> tokens) {
...
-        jsonFactory, new HttpRequestInitializer() {
-              @Override
-              public void initialize(HttpRequest request) throws IOException {
-                credential.initialize(request);
-              }
-            })
-        .setApplicationName(APPLICATION_NAME)
-        .build();
+  public static LanguageServiceClient createLanguageService() throws IOException{
+	  return LanguageServiceClient.create();
```
Comment (jerjou) [inline language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java]: This seems a little pedantic, no? Might be clearer to just use `LanguageServiceClient.create()` directly and just delete this method.

File: language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java [hunk 2]
```diff
@@ -188,41 +161,30 @@ public Analyze(CloudNaturalLanguage languageApi) {
...
-    AnalyzeSyntaxRequest request =
-        new AnalyzeSyntaxRequest()
-            .setDocument(new Document().setContent(text).setType("PLAIN_TEXT"))
-            .setEncodingType("UTF16");
-    CloudNaturalLanguage.Documents.AnalyzeSyntax analyze =
-        languageApi.documents().analyzeSyntax(request);
-    AnalyzeSyntaxResponse response = analyze.execute();
-    return response.getTokens();
+    AnalyzeSyntaxRequest request = AnalyzeSyntaxRequest.newBuilder()
+            .setDocument(Document.newBuilder().setContent(text).setType(Type.PLAIN_TEXT).build())
```
Comment (lesv) [inline language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java]: Line is way too long.

File: language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeIT.java [hunk 3]
```diff
@@ -85,11 +86,11 @@
         analyzeApp.analyzeSyntax(
             "President Obama was elected for the second term");
 
-    List<String> got = token.stream().map(e -> e.getPartOfSpeech().getTag())
+    List<Tag> got = token.stream().map(e -> e.getPartOfSpeech().getTag())
```
Comment (lesv) [inline language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeIT.java]: Yea - Java 8.   Good to see, but we don't typically use this in samples.  Note - I'm Ok w/ this.

File: language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java [hunk 4]
```diff
@@ -69,13 +71,22 @@
...
         ImmutableList.of(
-            new Entity().setName("Larry Page").setSalience(0.426f).setType("PERSON").setMetadata(
-                ImmutableMap.<String, String>builder()
-                    .put("knowledge_graph_mid", "/m/0gjpq")
-                    .put("wikipedia_url", "http://en.wikipedia.org/wiki/index.html?curid=60903")
-                    .build()),
-            new Entity().setName("search engine").setSalience(0.188f).setType("CONSUMER_GOOD"),
-            new Entity().setName("something"));
+            Entity.newBuilder().setName("Larry Page")
+                .setSalience(0.426f)
```
Comment (lesv) [inline language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java]: That seems really specific, should you explain it?

File: language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java [hunk 5]
```diff
@@ -69,13 +71,22 @@
...
+                .putAllMetadata(
+                    ImmutableMap.<String, String>builder()
+                        .put("knowledge_graph_mid", "/m/0gjpq")
+                        .put("wikipedia_url",
+                            "http://en.wikipedia.org/wiki/index.html?curid=60903")
+                        .build())
+                .build(),
+            Entity.newBuilder()
+                .setName("search engine")
+                .setSalience(0.188f)
```
Comment (lesv) [inline language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java]: Also seems very specific - does it deserve a comment?

File: language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java (hunk 2 above)
Comment (gguuss) [inline language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java]: Revised this line and adjacent lines to build document separate from request - makes all the lines a bit less lengthy.

File: language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeIT.java (hunk 3 above)
Comment (gguuss) [inline language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeIT.java]: Ack. Leaving as-is.

File: language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java (hunk 4 above)
Comment (gguuss) [inline language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java]: Done. Added verbose comments calling out this is mock data based on actual data.

File: language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java (hunk 5 above)
Comment (gguuss) [inline language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeTest.java]: Done. Unless, you mean to comment on the inner members, e.g.

    Entity.newBuilder()
        .setName("search engine") // The proper name for the mock entity
        .setSalience(0.188f) // The prominence of this entity relative to others in the document

I'm not sure how much detail is appropriate here in the mocks. For example, AFAICT the salience of entities sums up to 1. This means that if you sorted the entities by salience, descending, the most prominent entities in the target text will appear first: the actual salience values just tell you how important this particular entity is within the scope of the document.

**Pull request merged**